tools/install_tensorflow_cc.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+export JAVA_HOME=/export/b02/hxu/TensorFlow/java/jdk1.8.0_121
+PATH=$PATH:$PWD/bazel/output
+export HOME=$PWD/tensorflow_build/
+mkdir -p $HOME
+
+java=`which java`
+if [ $JAVA_HOME != "" ]; then
+  java=$JAVA_HOME/bin/java
+fi
+
+java_version=`$java -version 2>&1 | head -n 1 | awk '{print $3}' | sed s=\"==g`
+good_version=`echo 1.8 $java_version | awk '{if($1<$2)print 1; else print 0}'`
+if [ $good_version -eq 0 ]; then
+  echo You have jdk version = $java_version, which is older than 1.8
+  echo You need to download a later than 1.8 JDK version at
+  echo http://www.oracle.com/technetwork/pt/java/javase/downloads/jdk8-downloads-2133151.html
+  echo and set your JAVA_HOME to point to where it is installed
+  exit 1
+else
+  echo You have jdk version = $java_version, which is newer than 1.8. We will continue the installation
+fi
+
+
+[ ! -f bazel.zip ] && wget https://github.com/bazelbuild/bazel/releases/download/0.5.1/bazel-0.5.1-dist.zip -O bazel.zip
+mkdir -p bazel
+cd bazel
+unzip ../bazel.zip
+./compile.sh
+cd ../
+
+# now bazel is built
+git clone https://github.com/tensorflow/tensorflow
+cd tensorflow
+./configure
+
+tensorflow/contrib/makefile/download_dependencies.sh 
+bazel build -c opt --copt=-msse4.2 //tensorflow:libtensorflow.so
+bazel build -c opt --copt=-msse4.2 //tensorflow:libtensorflow_cc.so
+
+echo Building tensorflow completed. You will need to go to kaldi/src/ and do
+echo \"make\" under tensorflow/ and tfbin/ to generate the binaries
+
+exit
+
+# the following would utilize the highest optimization but might not work in a
```

Comment:
Using "exit" to comment code is very confusing.  Just comment the lines and remove the "exit".